sources/platform/actors/development/programming_interface/environment_variables.md

```diff
@@ -78,7 +127,17 @@ You can access environment variables in your code as follows:
 In Node.js, use the `process.env` object:
 
 ```js
-console.log(process.env.APIFY_USER_ID);
+import { Actor } from 'apify';
+
+await Actor.init();
+
+// get api_token
+const api_token = process.env.api_token
+
+// print api_token to console
+console.log(api_token);
```

Comment:
I would not recomemnd printing sensitive value and if it would be secret, it would only print `****` but if we replace it with something like `MYSQL_USER`, it would be ok